ups/pipe_supertask.table

```diff
@@ -1,4 +1,5 @@
 setupRequired(pipe_base)
+setupRequired(pipe_tasks)
```

Comment:
Indeed, right now it's hard requirement, `pipe_tasks` has to be in `PYTHONPATH`, otherwise cmdLineActivator will fail. When registry materializes we will remove this big dependency.